recipe/meta.yaml

```diff
@@ -13,16 +13,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 200
```

Comment:
See this [hackpad]( https://conda-forge.hackpad.com/BLAS-Numpy-Friends-86De62hoNdT ) for details.